modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/input_localized.js

```diff
@@ -230,6 +230,20 @@ AUI.add(
 					return '#' + namespace + id + '_' + languageId;
 				},
 
+				_moveDefaultLanguageFlagAtFirstPosition(defaultLanguageId) {
```

Comment:
(Grammar) Should this be `ToFirstPosition` (instead of `AtFirstPosition`)?

Reply:
Yeah, I'll change that, thanks